code_conventions.md

```diff
@@ -256,17 +314,16 @@ Adhere to the following coding principles
 
 Documentation
 ====
-[ TODO ]
+[TODO]
 
 We use [Doxygen](http://www.doxygen.org/) to generate documentation. Try to keep your documentation in doxygen style.
 
 Doxygen documentation should always be next to the declaration of the thing documented - in the header file.
 
-Here's a small example:
+Here's a small example for C/C++
 ~~~~~~~~~~~~~~~{.cpp}
 /*!
- * Doxygen style comments!
- *
+ * @brief Description about function
```

Comment:
doxygen will already mark the first line as brief in this example, or am I wrong?

Reply:
It won't